.github/ISSUE_TEMPLATE/4_documentation_change.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Documentation change request
+about: Report documentation issue or request documentation changes
+title: ''
+labels: documentation
+assignees: ''
+
+---
+
+**Summary**
```

Comment:
Please add a colon after your bolded headers and be consistent on capitalization: **Additional Details:**

Reply:
Done.